Extension/package.nls.json

```diff
@@ -11,7 +11,7 @@
     "c_cpp.command.configurationProviderSelect.title": "Change Configuration Provider...",
     "c_cpp.command.configurationEditJSON.title": "Edit Configurations (JSON)",
     "c_cpp.command.configurationEditUI.title": "Edit Configurations (UI)",
-    "c_cpp.command.selectDefaultCompiler.title": "Select Default Compiler",
+    "c_cpp.command.SelectDefaultCompiler.title": "Select Default Compiler",
```

Comment:
`c_cpp.command.SelectDefaultCompiler.title` needs to match the ID from package.json. So the "SelectDefaultCompiler" part needs to be lower case too.